### PR TITLE
php new entry for sqrt function

### PR DESCRIPTION
**Pull request by Judis07**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/sqrt/sqrt.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - 'Computer Science'
+  - 'Web Design'
+Tags:
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`sqrt()`** is a function that calculates the square root of a number. It takes a single paramter in the form of a number and returns the square root of that number.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/sqrt/sqrt.md:14]:
```suggestion
The **`sqrt()`** function calculates the square root of a number. It takes a single parameter in the form of a number and returns the square root of that number.
```

File: content/php/concepts/math-functions/terms/sqrt/sqrt.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Syntax
+
+```pseudo
+$result = sqrt($value)
+```
+
+- `$value` is the number whose square root is being calculated.
+- `$result` is the resultant square root value.
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/sqrt/sqrt.md:24]: You should note here that the result for a negative number is `NAN`

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Judis07 I did a first review and left a couple of comments.

File: content/php/concepts/math-functions/terms/sqrt/sqrt.md (hunk 2 above)
Comment (Judis07) [reply content/php/concepts/math-functions/terms/sqrt/sqrt.md:24]: I have added a note mentioning this case.

**Review by caupolicandiaz — COMMENTED** [review]
@Judis07 @SSwiniarski Overall, looks solid. Made a few direct edits. This is ready to merge

**Review by caupolicandiaz — APPROVED** [review]

**Pull request merged**